ecl/ConfTestMoreOperations.ecl

```diff
@@ -25,10 +25,6 @@ Upper  := Triangle.Upper;
 Lower  := Triangle.Lower;
 Side := Types.Side;
 
-// Override BlockDimensions defaults to allow test with smaller matrixes
-max_partition_size_or := 1000;
-override := #STORED('BlockDimensions_max_partition_size', max_partition_size_or);
-
 // TEST -- Multiply two matrixes and then solve for an original
```

Comment:
By removing the override and the WHEN, you are causing this test to run in a single partition, since the default partition size is quite large.  If this is what you intend, fine, but that makes it a single-node operation, and does not exercise much of the distributed processing .  Be sure that is what you intend.

Reply:
I included these back into the file.